ironfish-cli/src/commands/wallet/multisig/dkg/round3.ts

```diff
@@ -17,6 +17,10 @@ export class DkgRound3Command extends IronfishCommand {
       description: 'The name of the secret to use for decryption during DKG',
       required: true,
     }),
+    name: Flags.string({
```

Comment:
Would `accountName` be a better name for the flag?

Reply:
Yeah I'm ok with that. Updated